crates/swift-runtime/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+#![feature(link_args)]
+
+use std::ffi;
+
+extern "C" {
+    fn swift_demangle(
+        mangledName: *const u8,
+        mangledNameLength: usize,
+        outputBuffer: *mut u8,
+        outputBufferSize: *mut usize,
+        flags: u32,
+    ) -> *const i8;
+}
```

Comment:
This is a much better way to do it than the nonsense I've done in https://github.com/nvzqz/swift-bindgen/pull/9.